hack/lib/etcd.sh

```diff
@@ -144,38 +144,40 @@ kube::etcd::cleanup() {
 }
 
 kube::etcd::install() {
-  (
```

Comment:
this is de-indented and dropping the `()` subshell, then adding the two `export ...` lines

Reply:
we call `cd` later - that changes the working dir for the caller.  This seems broken

Reply:
yeah, I checked out a brand new tree at HEAD and I hit the path which returns from this function with `pwd` in the third_party directory

Reply:
bash is super wonky about exit codes and errexit and conditionals, but I am cooking something up

Reply:
https://github.com/kubernetes/kubernetes/pull/122470

Reply:
> we call cd later - that changes the working dir for the caller. This seems broken

agree. thanks for fixing.